spec/factories/invoice_items.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do 
+  factory :invoice_item, class: InvoiceItem do 
+    association :item
+    association :invoice
+    quantity { Faker::Number.between(from: 1, to: 10) }
+    unit_price { Faker::Number.between(from: 1, to: 100) }
+    status {["pending", "packaged", "shipped"].sample}
```

Comment:
should we think about whether invoice_item has a default status of pending when it is created?